src/dateManipulation.js

```diff
@@ -0,0 +1,8 @@
+exports.mapHourToDate = hour => {
+    const array = hour.split(':');
```

Comment:
If we want, we can turn this feature on - [check it](http://stackoverflow.com/a/17379714/1777334)